src/core/app_irc_event.c

```diff
@@ -1836,6 +1836,10 @@ static void irc_event_numeric(SircSession *sirc, int event,
                         _("URL of %1$s is: %2$s"), chan, msg);
                 break;
             }
+        case SIRC_RFC_RPL_UMODEIS:
+            {
+                break;
+            }
```

Comment:
This change is not needed if we put the call of `events->numeric` in the default case.